README.rst

```diff
@@ -2,19 +2,19 @@
 django-compressedlistfield
 ==========================
 
-django-compressedlistfield is a reusable Django field that stores a compressed,
-auto growing, list of integers in your models.
+django-compressedlistfield provides a Django model field which stores
+a compressed, auto growing, list of integers.
 
-It uses bz2 for compresing the list, but you don't have to worry about it as it
-silently takes care of serialization. To use, simply add the field to one of
-your models.
+It uses bz2 for compresing the list, but you don't have to worry about
+it as it transparently takes care of serialization. To use, simply add the
+field to one of your models.
```

Comment:
Besides the line changes (emacs fill-paragraph command), this is just s/silently/transparently/